### DISCUSSION TITLE
Identify profile handles by customized alias?

### DISCUSSION THREAD

**Discussion by est** [bluesky-social/atproto#2318] [General Discussion]
While many people put first & last names on their profile but some don't. In such case if we follow some stranger, can we give their handle a customized alias? (or even a short remark) Like how save phone numbers into contacts by giving a name.

Should this be done at atproto level or just a feature per hosting service?

**Comment by MasterJ93** [comment]
My guess is this is probably going to be part of the user's Follow record when they select someone they want to follow. However, records are public. So alternatively, there could be a section in the preferences that will link the alias with a Follow record. I guess the only issue left would be to sync the changes if the user deletes the record.
